index.html

```diff
@@ -848,12 +848,14 @@
   <dd>
     Because the [=user agent=] is a [=trustworthy agent=], it is held to be loyal to the
     [=person=] using it in all situations, including in preference to the [=user agent=]'s implementer.
-    When a [=user agent=] carries out [=processing=] that is not in the [=person=]'s
-    interest but instead benefits another [=actor=] (such as the user agent's implementer) that behaviour is
-    known as <dfn>self-dealing</dfn>. Behaviour can be [=self-dealing=] even if it is done at the
-    same time as [=processing=] that is in the [=person=]'s interest, what matters is that it
-    potentially conflicts with that [=person=]'s interest. [=Self-dealing=] is always
-    [=inappropriate=]. Loyalty is the avoidance of [=self-dealing=].
+    When a [=user agent=] carries out [=processing=] that is detrimental to a [=person=]'s
+    interests and instead benefits another [=actor=] (typically, but not necessarily, the user agent's
+    implementer) that behaviour is known as <dfn>self-dealing</dfn>. Behaviour can be [=self-dealing=] even
```

Comment:
```suggestion
    interests and instead benefits another [=actor=], this is <dfn>disloyal</dfn>. Often this would benefit the user agent itself, in which case it is known as <dfn>self-dealing</dfn>. Behaviour can be [=disloyal=] even
```